### PR TITLE
Removing the SyntaxVisualizers from Roslyn.sln

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @amcasey, @jmarolf

### CODE REVIEW

File: src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerDgmlHelper/SyntaxVisualizerDgmlHelper.vbproj [hunk 1]
```diff
@@ -18,21 +18,18 @@
...
-      <Project>{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}</Project>
-      <Name>CodeAnalysis</Name>
-      <Private>False</Private>
-    </ProjectReference>
-    <ProjectReference Include="..\..\..\..\Compilers\VisualBasic\Portable\BasicCodeAnalysis.vbproj">
-      <Project>{2523D0E6-DF32-4A3E-8AE0-A19BFFAE2EF6}</Project>
-      <Name>BasicCodeAnalysis</Name>
-      <Private>False</Private>
-    </ProjectReference>
+    <Reference Include="Microsoft.CodeAnalysis.CSharp">
```
Comment (amcasey) [inline src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerDgmlHelper/SyntaxVisualizerDgmlHelper.vbproj]: Were the project references pulling in other assemblies transitively?  Do those assemblies need to be directly referenced now?

**Comment by amcasey** [comment]
:+1:

**Comment by tannergooding** [comment]
FYI. @Pilchie

**Comment by Pilchie** [comment]
Won't the VSIX still try to build on our signed build, but not find these projects?

**Comment by tannergooding** [comment]
@Pilchie, the templates solution (which generates the sdk vsix containing these files) now holds these projects and builds them.

File: src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerDgmlHelper/SyntaxVisualizerDgmlHelper.vbproj (hunk 1 above)
Comment (tannergooding) [inline src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerDgmlHelper/SyntaxVisualizerDgmlHelper.vbproj]: Nothing failed when building locally.

Comment (Pilchie) [inline src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerDgmlHelper/SyntaxVisualizerDgmlHelper.vbproj]: Is the content of the sdk vsix the same?

**Comment by Pilchie** [comment]
Oh, I missed that change.  Great.

**Pull request merged**